data/json/items/generic.json

```diff
@@ -2439,28 +2439,19 @@
         "flags" : ["LITCIG", "TOBACCO", "LIGHT_4"]
     },
     {
-        "type" : "COMESTIBLE",
+        "type" : "GENERIC",
         "id" : "cig_butt",
         "name" : "cigarette butt",
         "weight" : 0,
         "color" : "dark_gray",
-        "addiction_type" : "nicotine",
-        "spoils_in" : 0,
-        "stim" : 1,
         "to_hit" : -2,
-        "comestible_type" : "MED",
         "symbol" : "!",
-        "quench" : 0,
-        "healthy" : -3,
-        "addiction_potential" : 50,
-        "nutrition" : 0,
         "description" : "What was once a wonderfully addictive tube of dried tobacco leaf is now just a smelly piece of trash.  What a tragedy!\nThe leftover tobacco in a few of these could probably be used to roll another cigarette.  If you're willing to go that far...",
         "price" : 0,
         "tool" : "null",
```

Comment:
Ok i just think that tool is have something with, crafting, and that is a crafting ingredient, 
So thanks for info.